app/controllers/application_controller.rb

```diff
@@ -1,2 +1,10 @@
 class ApplicationController < ActionController::API
+  skip_before_filter :verify_authenticity_token
+  before_filter :add_allow_credentials_headers
+
+  def add_allow_credentials_headers                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          
+    response.headers['Access-Control-Allow-Origin'] = request.headers['Origin'] || '*'                                                                                                                                                                                                     
+    response.headers['Access-Control-Allow-Credentials'] = 'true'                                                                                                                                                                                                                          
```

Comment:
Reading up on this, it seems it's used to indicate to the browser that it's OK to send cookies. I don't think we should be sending cookies with requests if we can help it.